src/Views/help.html

```diff
@@ -18,67 +18,55 @@ <h4 class="modal-title">Help</h4>
-                                <input class="form-control" id="title" ng-model="vm.freshdeskTicketData.subject" required>
+                                <input class="form-control" id="title" ng-model="vm.ticketData.subject" required>
                             </div>
                             <div class="form-group">
                                 <label for="message">Description:</label>
-                                <textarea class="form-control" rows="6" id="description" ng-model="vm.freshdeskTicketData.description" required></textarea>
+                                <textarea class="form-control" rows="6" id="description" ng-model="vm.ticketData.description" required></textarea>
                             </div>
-                            <div class="form-group">
+                            <!-- <div class="form-group">
                                 <label for="title">Upload an Attachment (optional):</label><br />
                                 <span class="btn-file">
-                                    <i class="fa fa-paperclip"></i>&nbsp;&nbsp;Attach File <input type="file" ngf-select ng-model="vm.freshdeskTicketData.attachment" name="file" ngf-accept="'image/*'">
+                                    <i class="fa fa-paperclip"></i>&nbsp;&nbsp;Attach File <input type="file" ngf-select ng-model="vm.ticketData.attachment" name="file" ngf-accept="'image/*'">
                                 </span>  
-                                <br /><span ng-show="vm.freshdeskTicketData.attachment"><i class="fa fa-check" style="color:green"></i>&nbsp;&nbsp;{{vm.freshdeskTicketData.attachment.name}}<button class="close removeItem" style="" ng-click="vm.freshdeskTicketData.attachment = null">&times;</button></span>
-                            </div>
+                                <br /><span ng-show="vm.ticketData.attachment"><i class="fa fa-check" style="color:green"></i>&nbsp;&nbsp;{{vm.ticketData.attachment.name}}<button class="close removeItem" ng-click="vm.ticketData.attachment = null">&times;</button></span>
+                            </div> -->
 
-                        <button type="submit" class="btn-wim-default" ng-click="vm.submitFreshDeskTicket(form.$valid)" ng-disabled="vm.submittingSupportTicket"><i class="fa fa-share"></i><span >&nbsp;&nbsp;Submit Support Request</span></button>
-                        <div class="wim-alert-primary" ng-show="vm.submittingSupportTicket">Your is being submitted, please be patient</div>
-                        <div class="wim-alert-primary" ng-show="vm.showSuccessAlert">Your support request has been successfully submitted.  A member of the Streamstats team will contact you as soon as possible.</div>
+                        <button type="submit" class="btn-wim-default" ng-click="vm.submitTicket(form.$valid)" ng-disabled="vm.submittingSupportTicket"><i class="fa fa-share"></i><span >&nbsp;&nbsp;Create Email</span></button>

```

Comment:
Huh strange, it was working on mine. I guess we could just have text that tells them to email?

Reply:
I think that might be best, unfortunately. Because it kind of seemed like it _did_ work for me, since I didn't know what to expect. I even checked to see if a ticket was indeed created haha.